Resources/Prototypes/_NF/Recipes/Lathes/weapons.yml

```
@@ -1,48 +1,43 @@
 # Ranged weapons
 - type: latheRecipe
   id: WeaponLaserPistolNF
+  parent: BaseWeaponRecipe
   result: WeaponLaserPistolNF
-  category: Weapons
-  completetime: 5
   materials:
     Steel: 1200
     Glass: 800
     Silver: 850
 
 - type: latheRecipe
   id: WeaponRevolverArgenti
+  parent: BaseWeaponRecipe
```

Did you want a change from 5 to 2 seconds?
```suggestion
  parent: BaseWeaponRecipeLong
```